source/world/level/levelgen/feature/LargeFeature.cpp

```diff
@@ -13,8 +13,8 @@ void LargeFeature::apply(ChunkSource* csrc, Level* level, int x, int z, uint8_t*
 {
 	m_random.setSeed(level->getSeed());
 
-	int r1 = 2 * (m_random.nextLong() / 2) + 1;
-	int r2 = 2 * (m_random.nextLong() / 2) + 1;
+	int r1 = 2 * (m_random.nextInt() / 2) + 1;
+	int r2 = 2 * (m_random.nextInt() / 2) + 1;
```

Comment:
Revert

Reply:
Why?

Reply:
Shouldn't we standardize these functions so that their integer lengths are made clear? nextLong() on 64-bit platforms will be 64-bits, but on 32-bit platforms it will be 32-bit. I see no point in making it potentially longer than it needs to be. I understand why we wouldn't want to do nextInt() on potentially very old platforms, since the integers could potentially be under 32-bits in size, but at that point, we don't we just use genrand_int32() all the time so the desired lengths are clearly defined?

Reply:
I clarified that nextLong() should return a uint32_t in an earlier review.

Reply:
So then what does nextInt() return?